lib/convenient_service/support/dependency_container/entities/method.rb

```diff
@@ -92,20 +102,40 @@ def define_in_module!(mod)
           def ==(other)
             return unless other.instance_of?(self.class)
 
-            return false if full_name != other.full_name
+            return false if slug != other.slug
             return false if scope != other.scope
             return false if body != other.body
+            return false if alias_slug != other.alias_slug
 
             true
           end
 
+          ##
+          # @return [String, Symbol]
+          #
+          def import_name
+            alias_slug || slug
+          end
+
+          ##
+          # @return [Hash]
+          #
+          def to_kwargs
+            {
+              slug: slug,
+              scope: scope,
+              body: body,
+              alias_slug: alias_slug
+            }
+          end
+
           private
 
           ##
           # @return [Array<String>]
           #
-          def full_name_parts
-            @full_name_parts ||= Utils::String.split(full_name, ".", "::").map(&:to_sym)
+          def slug_parts
```

Comment:
**Critical**

Please, create two separate methods `slug_parts` and `alias_slug_parts`.

It may be confusing that `slug_parts` sometimes returns `alias_slug_parts` as it is implemented right now.